test/lit/wat-kitchen-sink.wast

```diff
@@ -1180,6 +1188,86 @@
   return
  )
 
+ ;; CHECK:      (func $ref-is (type $anyref_=>_none) (param $0 anyref)
+ ;; CHECK-NEXT:  (drop
+ ;; CHECK-NEXT:   (ref.is_null
+ ;; CHECK-NEXT:    (local.get $0)
+ ;; CHECK-NEXT:   )
+ ;; CHECK-NEXT:  )
+ ;; CHECK-NEXT:  (drop
+ ;; CHECK-NEXT:   (ref.is_func
+ ;; CHECK-NEXT:    (local.get $0)
+ ;; CHECK-NEXT:   )
+ ;; CHECK-NEXT:  )
+ ;; CHECK-NEXT:  (drop
+ ;; CHECK-NEXT:   (ref.is_data
+ ;; CHECK-NEXT:    (local.get $0)
+ ;; CHECK-NEXT:   )
+ ;; CHECK-NEXT:  )
+ ;; CHECK-NEXT:  (drop
+ ;; CHECK-NEXT:   (ref.is_i31
+ ;; CHECK-NEXT:    (local.get $0)
+ ;; CHECK-NEXT:   )
+ ;; CHECK-NEXT:  )
+ ;; CHECK-NEXT: )
+ (func $ref-is (param anyref)
+  local.get 0
+  ref.is_null
+  drop
+  local.get 0
+  ref.is_func
+  drop
+  local.get 0
+  ref.is_data
+  drop
+  local.get 0
+  ref.is_i31
+  drop
+ )
+
+ ;; CHECK:      (func $ref-eq (type $eqref_=>_i32) (param $0 eqref) (result i32)
+ ;; CHECK-NEXT:  (ref.eq
+ ;; CHECK-NEXT:   (local.get $0)
+ ;; CHECK-NEXT:   (local.get $0)
+ ;; CHECK-NEXT:  )
+ ;; CHECK-NEXT: )
+ (func $ref-eq (param eqref) (result i32)
+  local.get 0
+  local.get 0
```

Comment:
With different locals the order could also be checked, but not sure if that's worth it. Once we switch by default there will be lots of testing of that, also.

Reply:
Oh right, I forgot that I had been trying to do that. Will update.